ring/replication_set.go

```diff
@@ -109,6 +109,15 @@ type DoUntilQuorumConfig struct {
 
 	// If non-nil, DoUntilQuorum will emit log lines and span events during the call.
 	Logger *spanlogger.SpanLogger
+
+	// If non-nil and a request returns an error for which IsTerminalError returns true,
```

Comment:
Does non-nil refer to error or to request?

Reply:
I'll clarify this - "If non-nil" at the start refers to `IsTerminalError` being non-nil.

Reply:
Resolved in fc2a12f